frozendict/__init__.py

```diff
@@ -1,6 +1,7 @@
 import collections
-import operator
-import functools
+
+
+iteritems = getattr(dict, 'iteritems', dict.items)
```

Comment:
@lurch  the idea was to avoid this kind of lookups *every time* we have to hash the `frozendict`. This is both a slight optimisation as well, but we can also implement it like this if we really want to keep it in scope, but there is no reason IMHO to do that.

```
def __hash__(self, iteritems=getattr(...)):
    ...
```